### DISCUSSION TITLE
Nbminer Random Worker name creation

### DISCUSSION THREAD

**Discussion by ekinsipahi** [NebuTech/NBMiner#784] [Q&A]
Hi I need to know how can I create random Worker name’s everytime I login  I Have 30pc’s. I want to Install nbminer to all of them but I want It to be fast Installations so how can I give random Worker names each new login for each pc? with a line of code ? Or do i have to put sth in the .bat file that starts the mining? Can someone help I will be really grateful if you guys can help🙏🏽

**Comment by Tom5051** [comment]
in your batch script, where you would set a worker name, try .worker%random%
it should give you the output of worker45394 for example. basically a 5 digit random number.
you can change worker to anything you want. %random% gives you the variable you want

**Reply by ekinsipahi** [reply]
If I want to add more than 5 numbers what should I add

**Comment by Tom5051** [comment]
%random% = 5 random numbers
%random%%random% = 10
etc etc
